R/initialization.R

```diff
@@ -4,82 +4,84 @@
+    eta_growth = abs(stats::rnorm(n_subpops, 0, stdev)),
+    sigma_growth = abs(stats::rnorm(1, 0, stdev)),
     autoreg_rt = abs(stats::rnorm(
       1,
-      autoreg_rt_a / (autoreg_rt_a + autoreg_rt_b),
+      params$autoreg_rt_a / (params$autoreg_rt_a + params$autoreg_rt_b),
       0.05
     )),
     log_r_mu_intercept = stats::rnorm(
       1,
-      convert_to_logmean(1, 0.1),
-      convert_to_logsd(1, 0.1)
+      convert_to_logmean(1, stdev),
+      convert_to_logsd(1, stdev)
     ),
     error_site = matrix(
       stats::rnorm(n_subpops * n_weeks,
         mean = 0,
-        sd = 0.1
+        sd = stdev
       ),
       n_subpops,
       n_weeks
     ),
     autoreg_rt_site = abs(stats::rnorm(1, 0.5, 0.05)),
     autoreg_p_hosp = abs(stats::rnorm(1, 1 / 100, 0.001)),
-    sigma_rt = abs(stats::rnorm(1, 0, 0.01)),
+    sigma_rt = abs(stats::rnorm(1, 0, stdev)),
     i0_over_n = stats::plogis(stats::rnorm(1, stats::qlogis(i0 / pop), 0.05)),
-    initial_growth = stats::rnorm(1, 0, 0.001),
+    initial_growth = stats::rnorm(1, 0, stdev),
     inv_sqrt_phi_h = 1 / sqrt(200) + stats::rnorm(1, 1 / 10000, 1 / 10000),
     sigma_ww_site_mean = abs(stats::rnorm(
-      1, sigma_ww_site_prior_mean_mean,
-      0.1 * sigma_ww_site_prior_mean_sd
+      1, params$sigma_ww_site_prior_mean_mean,
+      stdev * params$sigma_ww_site_prior_mean_sd
     )),
     sigma_ww_site_sd = abs(stats::rnorm(
-      1, sigma_ww_site_prior_sd_mean,
-      0.1 * sigma_ww_site_prior_sd_sd
+      1, params$sigma_ww_site_prior_sd_mean,
+      stdev * params$sigma_ww_site_prior_sd_sd
     )),
-    sigma_ww_site_raw = abs(stats::rnorm(n_ww_lab_sites, 0, 0.05)),
-    p_hosp_mean = stats::rnorm(1, stats::qlogis(p_hosp_mean), 0.01),
-    p_hosp_w = stats::rnorm(tot_weeks, 0, 0.01),
+    sigma_ww_site_raw = abs(stats::rnorm(n_ww_lab_sites, 0, stdev)),
+    p_hosp_mean = stats::rnorm(1, stats::qlogis(params$p_hosp_mean), stdev),
+    p_hosp_w = stats::rnorm(tot_weeks, 0, stdev),
     p_hosp_w_sd = abs(stats::rnorm(1, 0.01, 0.001)),
-    t_peak = stats::rnorm(1, t_peak_mean, 0.1 * t_peak_sd),
-    viral_peak = stats::rnorm(1, viral_peak_mean, 0.1 * viral_peak_sd),
+    t_peak = stats::rnorm(1, params$t_peak_mean, stdev * params$t_peak_sd),
+    viral_peak = stats::rnorm(
+      1, params$viral_peak_mean,
+      stdev * params$viral_peak_sd
+    ),
     dur_shed = stats::rnorm(
-      1, duration_shedding_mean,
-      0.1 * duration_shedding_sd
+      1, params$duration_shedding_mean,
+      stdev * params$duration_shedding_sd
     ),
-    log10_g = stats::rnorm(1, log10_g_prior_mean, 0.5),
-    ww_site_mod_raw = abs(stats::rnorm(n_ww_lab_sites, 0, 0.05)),
-    ww_site_mod_sd = abs(stats::rnorm(1, 0, 0.05)),
-    hosp_wday_effect = to_simplex(stats::rnorm(7, 1 / 7, 0.01)),
+    log10_g = stats::rnorm(1, params$log10_g_prior_mean, 0.5),
+    ww_site_mod_raw = abs(stats::rnorm(n_ww_lab_sites, 0, stdev)),
+    ww_site_mod_sd = abs(stats::rnorm(1, 0, stdev)),
```

Comment:
Smake on these two. It used to be 0.05, now it is 0.01.

Reply:
See https://github.com/CDCgov/ww-inference-model/pull/54/files#r1717397110